internal/pkg/aws/ecs/task.go

```diff
@@ -20,6 +20,11 @@ const (
 	shortTaskIDLength      = 8
 	shortImageDigestLength = 8
 	imageDigestPrefix      = "sha256:"
+
+	// These names are not defined as const in sdk.
+	// We will need to monitor if they have changed the key name in the future.
```

Comment:
I don't think they can ever do that as that would be a breaking change :) so I don't think we have to worry about it